CONTRIBUTING.md

```diff
@@ -65,3 +65,25 @@ CI runs GCS and inmem tests only for now. Not having these variables will produc
 7. You may merge the Pull Request in once you have the sign-off of at least one developers with write access, or if you
    do not have permission to do that, you may request the second reviewer to merge it for you.
 8. If you feel like your PR waits too long for a review, feel free to ping [`#thanos-dev`](https://join.slack.com/t/improbable-eng/shared_invite/enQtMzQ1ODcyMzQ5MjM4LWY5ZWZmNGM2ODc5MmViNmQ3ZTA3ZTY3NzQwOTBlMTkzZmIxZTIxODk0OWU3YjZhNWVlNDU3MDlkZGViZjhkMjc) channel on our slack for review!
+
+## Dependency management
+
+The Thanos project uses [Go modules](https://golang.org/cmd/go/#hdr-Modules__module_versions__and_more) to manage dependencies on external packages. This requires a working Go environment with version 1.11 or greater, git and bzr installed.
```

Comment:
can you add links to `bzr` download please

Reply:
And add the check to make :+1:

Reply:
Is it really nescessary, if you know how to install git, you probably don't need directions to install bzr.

Reply:
its more of a ... "lets make it easy for people to do the right thing"

Reply:
http://wiki.bazaar.canonical.com/Download

Reply:
Done